live-examples/js-examples/date-setmilliseconds.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">var event = new Date('Tue, August 19, 1975 23:15:30.123');
+
+console.log(event.getMilliseconds());
+// expected output: 123
+
+event.setMilliseconds(456);
+
+console.log(event.getMilliseconds());
+// expected output: 456
```

Comment:
I get `NaN` for these.

Reply:
Yes - same issues in Firefox and Safari regarding specifying a date with milliseconds.

Reply:
This one still isn't fixed.